From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MartijnMuijsers <martijnmuijsers@live.nl>
Date: Thu, 13 Oct 2022 20:55:09 +0200
Subject: [PATCH] Suki configuration

Original code by SuCraft, licensed under GPL-3.0.
You can find the original code on https://github.com/SuCraft/Suki

diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index a7e2fab3e174fe2c1d79d904f7eaff88dd0c3c82..b36419e0512b4b7ef421b54ca27db37e02fbe86e 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -1981,6 +1981,13 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
         }
         // Purpur end
 
+        // Suki start - Suki configuration
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration getSukiConfig() {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // Suki end - Suki configuration
+
         /**
          * Sends the component to the player
          *
